### PR TITLE
CI for Node

### PR DESCRIPTION
**Pull request by abgrund1**
commit-1-to-new-branch

### CODE REVIEW

**Comment by abgrund1** [comment]
CI for Node

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build --if-present
```
Comment (abgrund1) [inline .github/workflows/node.js.yml:28]: - run: npm run build --passWithNoTests

**Comment by abgrund1** [comment]
123

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - run: npm run build --passWithNoTests
```
Comment (kei1ne1ah1nung) [inline .github/workflows/node.js.yml:29]: 123

**Pull request closed without merging**